mcs/class/Microsoft.Build.Tasks/Test/Microsoft.Build.Tasks/CopyTest.cs

```diff
@@ -301,7 +301,8 @@ public void TestCopy_ReadOnlyUpdate ()
 				Assert.Fail ("Build failed");
 			}
 			Assert.IsTrue (File.Exists (target_file), "A2");
-			Assert.AreEqual (FileAttributes.Normal, File.GetAttributes (target_file), "A3");					
+			var target_file_attrs = File.GetAttributes (target_file);
+			Assert.IsTrue (FileAttributes.Normal == target_file_attrs || FileAttributes.Archive == target_file_attrs, "A3");
```

Comment:
This will just print "Expected: true, Actual: false" when it fails which makes it pretty useless. Can you please add target_file_attrs to the message?

Might also make sense to add a small comment in which cases the `Archive` attribute is set on Windows and possibly only do the check there.